src/Microsoft.Health.Abstractions/Microsoft.Health.Abstractions.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>Abstractions used by Microsoft Health.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Are there other projects that we could change to Standard too? It doesn't seem like a bad direction for Share Components

Reply:
Perhaps! But it can become difficult to use standard here if they use the microsoft extension libs. Let's say a lib wants to target standard and uses the latest 6.0.0 version of `Microsoft.Extensions.Logging`. However, it this lib is also used by our packages (or apps) that use ASP.NET Core, a .net 5 project would use the 5.0.0 of these packages. This makes it all a bit hairy.